src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,72 @@
+import clsx from "clsx";
+import React from "react";
+import DropdownIcon from "./DropdownIcon";
+
+export type DropdownItem = {
+  label: string;
+  href?: string;
+  disabled?: boolean;
+  value: string;
+  selected?: boolean;
+};
+
+type DropdownProps = {
+  list: DropdownItem[];
+  ariaLabel: string;
+  arrowIcon: "triangles" | "chevron";
+  classNames?: string;
+  innerClassNames?: { select?: string; option?: string; arrowWrapper?: string };
+  handleOnChange?: (e: React.ChangeEvent<HTMLSelectElement>) => void;
+  placeholder?: string;
+};
+
+const Dropdown: React.FunctionComponent<DropdownProps> = ({
+  arrowIcon,
+  ariaLabel,
+  list,
+  classNames,
+  innerClassNames,
+  handleOnChange,
+  placeholder
+}) => {
+  const classes = {
+    root: clsx("dropdown", classNames),
+    select: clsx("dropdown__select", innerClassNames?.select),
+    option: clsx("dropdown__option", innerClassNames?.option),
+    arrowWrapper: clsx("dropdown__arrows", innerClassNames?.arrowWrapper)
+  };
+  return (
+    <div className={classes.root}>
+      <select
+        className={classes.select}
+        aria-label={ariaLabel}
+        onChange={handleOnChange ? (e) => handleOnChange(e) : undefined}
+      >
+        {placeholder && (
+          <option hidden selected>
+            {placeholder}
+          </option>
+        )}
```

Comment:
Please reconsider the `hidden` and `selected` attributes.

1. As far as I can tell [`hidden` is not a valid attribute for `<option>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option). Do we intend it to be `disabled`?
2. The placeholder element should only be `selected` if no other element is selected. Only a single element may be selected if the select element does use `multiple`.